OtripleS.Web.Api.Tests.Unit/Services/Foundations/Teachers/TeacherServiceTests.Validations.Add.cs

```diff
@@ -140,6 +140,46 @@ public async void ShouldThrowValidationExceptionOnCreateWhenTeacherIsNullAndLogI
             this.storageBrokerMock.VerifyNoOtherCalls();
         }
 
+        [Fact]
+        public async void ShouldThrowValidationExceptionOnAddwhenTeacherStatusIsInvalidAndLogItAsync()
+        {
+            //given
+            DateTimeOffset date = GetRandomDateTime();
+            Teacher randomTeacher =  CreateRandomTeacher(date);
+            var invalidTeacher = randomTeacher;
+            invalidTeacher.Status = GetInValidTeacherStatus();
+            var invalidTeacherException = new InvalidTeacherException();
+                
+                invalidTeacherException.AddData(
+                    key:nameof(Teacher.Status),
```

Comment:
Do SHIFT + TAB here it will take this block of code back by one

Reply:
This has been fixed